### ISSUE TITLE
[Bug]: Seeding the database fails

### ISSUE THREAD

**Issue by PhantomKnight287** [typehero/typehero#970]
### What happened?

I am following the instructions provided in [Local.md](https://github.com/typehero/typehero/blob/main/LOCAL.md). When running `pnpm db:seed` the prisma throws an error saying `Unique constraint failed on the (not available)`

### Relevant log output

```zsh
PS G:\builds\OSS\typehero> pnpm db:seed

> typehero@0.1.0 db:seed G:\builds\OSS\typehero
> turbo db:seed

╭────────────────────────────────────────────────────────────────────────╮
│                                                                        │
│                  Update available v1.10.13 ≫ v1.10.16                  │
│    Changelog: https://github.com/vercel/turbo/releases/tag/v1.10.16    │
│               Run "npx @turbo/codemod update" to update                │
│                                                                        │
│      Follow @turborepo for updates: https://twitter.com/turborepo      │
╰────────────────────────────────────────────────────────────────────────╯
• Packages in scope: @repo/auth, @repo/db, @repo/emails, @repo/lint, @repo/monaco, @repo/og-image, @repo/tailwind, @repo/tsconfig, @repo/ui, admin, scripts, web
• Running db:seed in 12 packages
• Remote caching disabled
@repo/db:db:push: cache bypass, force executing e4772249d7687499
@repo/db:db:push:
@repo/db:db:push: > @repo/db@0.1.1 db:push G:\builds\OSS\typehero\packages\db
@repo/db:db:push: > dotenv -e ../../.env prisma db push
@repo/db:db:push:
@repo/db:db:push: Prisma schema loaded from schema.prisma
@repo/db:db:push: Datasource "db": MySQL database "phantomknight" at "aws.connect.psdb.cloud"
@repo/db:db:push: 
@repo/db:db:push: The database is already in sync with the Prisma schema.
@repo/db:db:push: 
✔ Generated Prisma Client (5.0.0 | library) to .\..\..\node_modules\@prisma\clie
@repo/db:db:push: nt in 338ms
@repo/db:db:push: 
@repo/db:db:seed: cache bypass, force executing 5a215341bf356ad1
@repo/db:db:seed: 
@repo/db:db:seed: > @repo/db@0.1.1 db:seed G:\builds\OSS\typehero\packages\db
@repo/db:db:seed: > dotenv -e ../../.env tsx ./seed/dev.ts
@repo/db:db:seed: 
@repo/db:db:seed: PrismaClientKnownRequestError: 
@repo/db:db:seed: Invalid `prisma.challenge.createMany()` invocation in
@repo/db:db:seed: g:\builds\OSS\typehero\packages\db\seed\dev.ts:29:26
@repo/db:db:seed: 
@repo/db:db:seed:   26   })
@repo/db:db:seed:   27   .then((r) => r.map((u) => u.id));
@repo/db:db:seed:   28 
@repo/db:db:seed: → 29 await prisma.challenge.createMany(
@repo/db:db:seed: Unique constraint failed on the (not available)
@repo/db:db:seed:     at Object.handleRequestError (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:8308)
@repo/db:db:seed:     at Object.handleAndLogRequestError (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:7697)
@repo/db:db:seed:     at Object.request (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:7307)
@repo/db:db:seed:     at <anonymous> (g:\builds\OSS\typehero\packages\db\seed\dev.ts:29:3) {
@repo/db:db:seed:   code: 'P2002',
@repo/db:db:seed:   clientVersion: '5.0.0',
@repo/db:db:seed:   meta: { target: null }
@repo/db:db:seed: }
@repo/db:db:seed:  ELIFECYCLE  Command failed with exit code 1.
@repo/db:db:seed: ERROR: command finished with error: command (G:\builds\OSS\typehero\packages\db) pnpm run db:seed exited (1)
@repo/db#db:seed: command (G:\builds\OSS\typehero\packages\db) pnpm run db:seed exited (1)

 Tasks:    1 successful, 2 total
Cached:    0 cached, 2 total
  Time:    19.265s 
Failed:    @repo/db#db:seed

Error: experimentalSpaceId is enabled, but repo is not linked to API. Run `turbo link` or `turbo login` first
 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by bautistaaa** [comment]
can you try changing the command from 
```
    "refresh": "turbo db:reset db:seed --concurrency=1",
```

```
 "refresh": "turbo db:reset && turbo db:seed",
```

if this works can you PR these changes

**Comment by PhantomKnight287** [comment]
This doesn't fix the problem. I tried it on a locally hosted mysql instance and it is working properly. I tried it on a freshly created planetscale instance and the error I got was `ConnectorError`

```bash
PS G:\builds\OSS\typehero> pnpm refresh

> typehero@0.1.0 refresh G:\builds\OSS\typehero
> turbo db:reset && turbo db:seed

╭────────────────────────────────────────────────────────────────────────╮
│                                                                        │
│                  Update available v1.10.13 ≫ v1.10.16                  │
│    Changelog: https://github.com/vercel/turbo/releases/tag/v1.10.16    │
│               Run "npx @turbo/codemod update" to update                │
│                                                                        │
│      Follow @turborepo for updates: https://twitter.com/turborepo      │
╰────────────────────────────────────────────────────────────────────────╯
• Packages in scope: @repo/auth, @repo/db, @repo/emails, @repo/lint, @repo/monaco, @repo/og-image, @repo/tailwind, @repo/tsconfig, @repo/ui, admin, scripts, web
• Running db:reset in 12 packages
• Remote caching disabled
@repo/db:db:reset: cache bypass, force executing 4550f5262dcd3f48
@repo/db:db:reset: 
@repo/db:db:reset: > @repo/db@0.1.1 db:reset G:\builds\OSS\typehero\packages\db
@repo/db:db:reset: > dotenv -e ../../.env prisma db push -- --force-reset
@repo/db:db:reset:
@repo/db:db:reset: Prisma schema loaded from schema.prisma
@repo/db:db:reset: Datasource "db": MySQL database "phantom" at "us-east.connect.psdb.cloud"
@repo/db:db:reset: 
@repo/db:db:reset: The MySQL database "phantom" at "us-east.connect.psdb.cloud" was successfully reset.
@repo/db:db:reset: 
@repo/db:db:reset: Your database is now in sync with your Prisma schema. Done in 10.71s
@repo/db:db:reset:
✔ Generated Prisma Client (5.0.0 | library) to .\..\..\node_modules\@prisma\clie
@repo/db:db:reset: nt in 275ms
@repo/db:db:reset:

 Tasks:    1 successful, 1 total
Cached:    0 cached, 1 total
  Time:    24.7s

Error: experimentalSpaceId is enabled, but repo is not linked to API. Run `turbo link` or `turbo login` first
╭────────────────────────────────────────────────────────────────────────╮
│                                                                        │
│                  Update available v1.10.13 ≫ v1.10.16                  │
│    Changelog: https://github.com/vercel/turbo/releases/tag/v1.10.16    │
│               Run "npx @turbo/codemod update" to update                │
│                                                                        │
│      Follow @turborepo for updates: https://twitter.com/turborepo      │
╰────────────────────────────────────────────────────────────────────────╯
• Packages in scope: @repo/auth, @repo/db, @repo/emails, @repo/lint, @repo/monaco, @repo/og-image, @repo/tailwind, @repo/tsconfig, @repo/ui, admin, scripts, web
• Running db:seed in 12 packages
• Remote caching disabled
@repo/db:db:push: cache bypass, force executing 44bb18cda071dc2b
@repo/db:db:push: 
@repo/db:db:push: > @repo/db@0.1.1 db:push G:\builds\OSS\typehero\packages\db
@repo/db:db:push: > dotenv -e ../../.env prisma db push
@repo/db:db:push:
@repo/db:db:push: Prisma schema loaded from schema.prisma
@repo/db:db:push: Datasource "db": MySQL database "phantom" at "us-east.connect.psdb.cloud"
@repo/db:db:push: 
@repo/db:db:push: The database is already in sync with the Prisma schema.
@repo/db:db:push:
✔ Generated Prisma Client (5.0.0 | library) to .\..\..\node_modules\@prisma\clie
@repo/db:db:push: nt in 344ms
@repo/db:db:push:
@repo/db:db:seed: cache bypass, force executing 2a0b3d127d9dffeb
@repo/db:db:seed: 
@repo/db:db:seed: > @repo/db@0.1.1 db:seed G:\builds\OSS\typehero\packages\db
@repo/db:db:seed: > dotenv -e ../../.env tsx ./seed/dev.ts
@repo/db:db:seed:
@repo/db:db:seed: PrismaClientUnknownRequestError: 
@repo/db:db:seed: Invalid `prisma.challenge.update()` invocation in
@repo/db:db:seed: g:\builds\OSS\typehero\packages\db\seed\dev.ts:96:26
@repo/db:db:seed:
@repo/db:db:seed:   93   replies.push(createComment(++commentNum, comment.id), createComment(++commentNum, comment.id));
@repo/db:db:seed:   94 }
@repo/db:db:seed:   95
@repo/db:db:seed: → 96 await prisma.challenge.update(
@repo/db:db:seed: Error occurred during query execution:
@repo/db:db:seed: ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError { code: 1105, message: "target: phantom.-.primary: vttablet: rpc error: code = Aborted desc = transaction 1698499210184552137: ended at 2023-10-28 13:28:49.010 UTC (exceeded timeout: 20s) (CallerID: ajjl8z2en1c9n9ofh5oo)", state: "HY000" })), transient: false })
@repo/db:db:seed:     at Object.handleRequestError (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:8480)
@repo/db:db:seed:     at Object.handleAndLogRequestError (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:7697)
@repo/db:db:seed:     at Object.request (g:\builds\OSS\typehero\node_modules\@prisma\client\runtime\library.js:122:7307)
@repo/db:db:seed:     at <anonymous> (g:\builds\OSS\typehero\packages\db\seed\dev.ts:96:3) {
@repo/db:db:seed:   clientVersion: '5.0.0'
@repo/db:db:seed: }
@repo/db:db:seed:  ELIFECYCLE  Command failed with exit code 1.
@repo/db:db:seed: ERROR: command finished with error: command (G:\builds\OSS\typehero\packages\db) pnpm run db:seed exited (1)
@repo/db#db:seed: command (G:\builds\OSS\typehero\packages\db) pnpm run db:seed exited (1)

 Tasks:    1 successful, 2 total
Cached:    0 cached, 2 total
  Time:    1m56.563s
Failed:    @repo/db#db:seed

Error: experimentalSpaceId is enabled, but repo is not linked to API. Run `turbo link` or `turbo login` first
 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by bautistaaa** [comment]
unsure what that error means tbh. im not a windows user either but there have been a couple that had issues and resolved them in the discord. id suggest joining and asking them.
